lxd/storage/drivers/driver_powerflex.go

```diff
@@ -0,0 +1,208 @@
+package drivers
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/canonical/lxd/lxd/migration"
+	"github.com/canonical/lxd/lxd/operations"
+	"github.com/canonical/lxd/shared"
+	"github.com/canonical/lxd/shared/api"
+	"github.com/canonical/lxd/shared/validate"
+)
+
+// powerFlexDefaultUser represents the default PowerFlex user name.
+const powerFlexDefaultUser = "admin"
+
+// powerFlexDefaultSize represents the default PowerFlex volume size.
+const powerFlexDefaultSize = "8GiB"
+
+var powerFlexLoaded bool
+var powerFlexVersion string
+
+type powerflex struct {
+	common
+
+	client *powerFlexClient
+}
+
+// load is used to run one-time action per-driver rather than per-pool.
+func (d *powerflex) load() error {
+	// Done if previously loaded.
+	if powerFlexLoaded {
+		return nil
+	}
+
+	// Detect and record the version.
+	out, err := shared.RunCommand("nvme", "version")
+	if err != nil {
+		return err
+	}
+
+	fields := strings.Split(strings.TrimSpace(out), " ")
+	if strings.HasPrefix(out, "nvme version ") && len(fields) > 2 {
+		powerFlexVersion = fmt.Sprintf("%s (nvme-cli) / x.y.z (sdc)", fields[2])
```

Comment:
Lets lose the sdc part until such time as sdc is added (in a separate PR).

Reply:
Removed all logic required to implement the SDC. Some data structures still show SDC since this is how they are named internally in PowerFlex.

Reply:
I left the `(nvme-cli)` suffix so that it is clear which version is displayed here. Unlike for the others (e.g. `ceph`) it might not be clear which version is meant here.